<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>treeInspector</class>
 <widget class="QWidget" name="treeInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tree Inspector</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>931</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlayMain">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="layTree"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="tblNodeInfo"/>
      </item>
      <item>
       <widget class="QLabel" name="lblImage">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
